tinyagentos/routes/agent_auth_requests.py

```
@@ -167,6 +179,30 @@ async def create_auth_request(request: Request, body: CreateAuthRequest):
+                title="Access request",
+                message=f"{record['identity_claim']} is requesting {', '.join(scopes)}",
+                level="info",
+                source="auth_requests",
+                data={
+                    "request_id": record["id"],
+                    "identity_claim": record["identity_claim"],
+                    "framework": record["framework"],
+                    "requested_scopes": list(scopes),
+                },
+            )
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)